src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -534,22 +534,15 @@ ss::future<cluster::init_tm_tx_reply> tx_gateway_frontend::do_init_tm_tx(
     auto maybe_tx = co_await stm->get_actual_tx(tx_id);
 
     if (!maybe_tx.has_value()) {
-        if (maybe_tx.error() == tm_stm::op_status::not_leader) {
-            vlog(
-              clusterlog.trace,
-              "this node isn't a leader for tx.id={} coordinator",
-              tx_id);
-            co_return init_tm_tx_reply{.ec = tx_errc::not_coordinator};
-        }
```

Comment:
I think you added this block of code in the first commit? if so, then you could drop the hunk from both commits.

Reply:
oops, it's a spill over from an another commit, removing it, thanks!